sushi-config.yaml

```diff
@@ -61,7 +61,7 @@ pages:
   artefacts.md:
     title: FHIR artefacts      
     profiles-and-extensions.md:
```

Comment:
add to index.md under FHIR artefacts the entry for capability statement

Reply:
Fixed.